theme/layouts/_navigation.ejs

```diff
@@ -1,17 +1,48 @@
-<nav id="main-navigation">
+<%
+const menuItems = ["", "documentation", "tutorial", "ready-projects", 
+                   "what's-new"];
+const query = site.queryPages;
+
+function getSubPages(nav)
+{
+  const subPages = query(({originalPathname}) => {
+    const parentPath = `${nav.originalPathname}/`;
+    return originalPathname.startsWith(parentPath) &&
+           !(originalPathname.substring(parentPath.length).includes("/"));
+  });
+  if (subPages.length) {
+    if (!nav.sub_items){
+      nav.sub_items = [];
+      for (const subPage of subPages) {
+        nav.sub_items.push(subPage);
+        getSubPages(subPage);
+      }
```

Comment:
Currently there is no order in which the items appear in the navigation, in order to fix that we can introduce a weight, so we can sort them out, for example:

```
---
title: Change
weight: 5
---
```